EAD-Session-6_1-Assignment/src/com/metacube/assignment/facade/InventoryFacade.java

```diff
@@ -0,0 +1,53 @@
+package com.metacube.assignment.facade;
+
+import java.util.List;
+import com.metacube.assignment.dao.BaseDao;
+import com.metacube.assignment.database.QueriesClass;
+import com.metacube.assignment.enums.Status;
+import com.metacube.assignment.factory.Factory;
+import com.metacube.assignment.model.Inventory;
+
+/**
+ * This class represents the facade layer of the software
+ * @author Jugal
+ * Dated 09/04/2019
+ */
+public class InventoryFacade {
+
+	BaseDao<Inventory> baseDao = Factory.getSQLDAO();
+	
+	//Method for POST
+	public Status insertItemToInventory(String name, int quantity)
+	{
+		return baseDao.insertItemToInventory(QueriesClass.postItemToInventory(name, quantity));
+	}
+	
+	//Method for GET
```

Comment:
Can describe more about function in the comments. Since by name of function its functionality is understandable, can avoid comment in this case.

Reply:
Ok